docs/multicluster/policy-only-mode.md

```diff
@@ -0,0 +1,83 @@
+# Antrea Multi-cluster with NetworkPolicy Only Mode
+
+Multi-cluster Gateway works with Antrea `networkPolicyOnly` mode, in which
+cross-cluster traffic is routed by Multi-cluster Gateways of member clusters,
+and the traffic goes through Antrea overlay tunnels between Gateways and local
+cluster Pods. Pod traffic within a cluster is still handled by the primary CNI,
+not Antrea.
+
+## Deploying Antrea in `networkPolicyOnly` mode with Multi-cluster feature
+
+This section describes steps to deploy Antrea in `networkPolicyOnly` mode
+with the Multi-cluster feature enabled on an EKS cluster.
+
+Assume you already have an EKS cluster, and have the `KUBECONFIG` environment
```

Comment:
We do not really need KUBECONFIG, etc. in this doc? I feel this paragraph is useless. Maybe we just remove it.